janusgraph-cql/src/test/java/org/janusgraph/JanusGraphCassandraContainer.java

```diff
@@ -140,6 +155,11 @@ public JanusGraphCassandraContainer(boolean bindDefaultPort) {
         }
     }
 
+    private void applyCassandraServerConfiguration(String resourcePath){
+        withClasspathResourceMapping(resourcePath, "/etc/cassandra/cassandra.yaml", BindMode.READ_WRITE);
```

Comment:
Just for my own understanding: These two lines will mount the config file into `/etc/cassandra/cassandra.yaml` AND `/etc/scylla/scylla.yaml` for the container, irrespective of whether it's Cassandra or Scylla, so each service can use its config file in the expected location, right?
And you probably did it like this because you wanted to avoid the complexity of having to distinguish again between Scylla and Cassandra only to save one more mount that isn't necessary?

If yes, then we can of course keep it like this. I'm really just asking to better understand this. (But maybe a small comment would help here in case anyone else later wonders about this.)

Reply:
Yes, initially I just decided to add the same config to both places for Cassandra and for Scylla even so one will be redundant because the `JanusGraphCassandraContainer` is for testing purposes, so it shouldn't be a problem to have redundant configs there.
That said, as you mentioned it I realized that it makes sense to simply reuse `isScylla()` method here and mount only one configuration depending on the container type. I will change this method here.